src/utils/authentication-util.ts

```diff
@@ -1,27 +1,41 @@
+import Cookies from "js-cookie";
+
 export class AuthenticationUtil {
   private static readonly USER = "user";
   private static readonly USER_TOKEN = "user-token";
 
+  // Attributes for cookies
+  private static readonly ATTRS = {
+    expires: 1,
```

Comment:
why 1? Does it mean "never"?

Reply:
1 means days. the js-cookie library you can enter the number of days or set a date when the cookie has expired
[js-cookie#expires](https://github.com/js-cookie/js-cookie#expires)

Reply:
Perhaps a constant that describes what this magic number is would be good

Reply:
very good idea :D